api/src/main/java/org/apache/iceberg/actions/RewriteManifests.java

```diff
@@ -44,6 +45,16 @@ public interface RewriteManifests
    */
   RewriteManifests rewriteIf(Predicate<ManifestFile> predicate);
 
+  /**
+   * Rewrite manifests in a given order, based on partition columns
+   *
+   * <p>If not set, manifests will be rewritten in the order of the table's spec.
```

Comment:
"in the order of the transforms in the table's current partition spec"

Reply:
Ack